.github/workflows/code-coverage.yml

```diff
@@ -0,0 +1,39 @@
+# Any coverage.xml will be picked up by this Github Action
+# Just make sure each coverage.xml is in a different folder
+name: Code Coverage
+
+on: push
+concurrency:
+  group: codecov-${{ github.workflow }}-${{ github.ref == 'refs/heads/master' && github.run_number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  codecov:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+          architecture: x64
+      - name: Setup Java 11
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'zulu'
+          java-version: '11'
+
+      - name: Checkout TorchServe
+        uses: actions/checkout@v2
+      - name: Install dependencies
+        run: |
+          python ts_scripts/install_dependencies.py --environment=dev
```

Comment:
@msaroufim do we cover this for GPU as well?

Reply:
Yeah I remember we have one test skipped on CPU. Given that our GPUs are more limited and this is a single test we can just skip. If there's a lot more GPU specific code in Python then I can change this PR or revisit